config/routes.rb

```diff
@@ -1,5 +1,20 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-  get '/a' => 'application#root'
+  root :to => 'restaurant#index'
+
+  resources :restaurant do
+    member do
+      get :delete
```

Comment:
you don't need to create a delete action via a get request since when you wrote `resources :restaurants` you get a "destroy" action via a "delete".

Same comment for cuisines.

Reply:
I have a specific view for delete, so routing is a must.